dev-haskell/beam-automigrate/metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>haskell@gentoo.org</email>
+		<name>Gentoo Haskell</name>
+	</maintainer>
+	<use>
+		<flag name="ghcipretty">Enable pretty-show for pretty-printing purposes</flag>
+		<flag name="integration-tests">Enable integration tests that talks to a Postgres DB.</flag>
+		<flag name="werror">Enable -Werror during development</flag>
```

Comment:
Of these, I would probably just keep the `ghcipretty` flag.

`integration-tests` sounds like something that we can't do with `FEATURES="network-sandbox"` (have you tried running the test?)

`werror` sounds like something that is only useful during development.

Reply:
Good call on removing the `werror` flag. It's been there on beam since I started making packages for it, but I've always left it in. I have no intention of enabling it and see no reason for others to either.

Reply:
`werror` has been removed from all these packages

Reply:
`integration-tests` indeed did not work and has been removed as well